src/java/org/apache/cassandra/gms/Gossiper.java

```diff
@@ -272,6 +278,25 @@ private static long getVeryLongTime()
         return 259200 * 1000; // 3 days
     }
 
+    private static long getFailedBootstrapTimeout()
+    {
+        String newtimeout = CassandraRelevantProperties.FAILED_BOOTSTRAP_TIMEOUT.getString();
+        if (newtimeout != null)
+        {
+            long longValue = Long.parseLong(newtimeout);
```

Comment:
why not just call `getLong(-1)`?  These properties expose the parsing in a generic way